src/PSDocs.Azure/docs/Azure.Template.Doc.ps1

```diff
@@ -20,15 +20,18 @@ function global:GetTemplateParameter {
                 Description = ''
                 DefaultValue = $Null
                 AllowedValues = $Null
+                Required = "Required"
             }
             if ([bool]$property.Value.PSObject.Properties['metadata'] -and [bool]$property.Value.metadata.PSObject.Properties['description']) {
                 $result.Description = $property.Value.metadata.description;
             }
             if ([bool]$property.Value.PSObject.Properties['defaultValue']) {
                 $result.DefaultValue = $property.Value.defaultValue;
+                $result.Required = "Optional"
```

Comment:
@vicperdana Same as above. Maybe this can be localized. Also I thought this was easy to read [example](https://github.com/Azure/azure-quickstart-templates/tree/master/modules/Microsoft.KeyVault/vaults/1.0) with a simple yes/ no. But happy with required/ optional.

Reply:
Updated this with Yes / No.